app/src/androidTest/assets/test.html

```diff
@@ -9,8 +9,6 @@
 <h1>focus test page</h1>
 <!-- More random words that should not end up on disk -->
 <p>groovy rabbits</p>
-<img id="rabbitImage" src="rabbit.jpg" alt="Smiley face" width="251" height="201">
```

Comment:
Hey, why did you remove that? It's used in my test to make sure the image does not remain on disk.